### PR TITLE
add test2.txt

### PR DESCRIPTION
**Pull request by xioota**
Plz check and pull this!

### CODE REVIEW

File: test2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+test2
```
Comment (kuteken) [inline test2.txt]: @xioota 
Fix to `test!!`

Comment (xioota) [inline test2.txt]: @kuteken Plz check it

Comment (kuteken) [inline test2.txt]: OK

**Comment by kuteken** [comment]
Reviewed. I'll merge it.

**Pull request merged**